tests/test_populate.py

```diff
@@ -0,0 +1,29 @@
+from tests.helpers import get_asset
+
+
+def test_populate(app, client, requests_mock, mocker):
+    with app.app_context():
+        gateway = 'http://api-gateway'
+        es_url = 'http://elasticsearch'
+        path_to_index = '/scholarly-articles'
+        app.config['GATEWAY_URL'] = gateway
+        app.config['ELASTICSEARCH_HOSTS'] = [es_url]
+        app.config['PATHS_TO_INDEX'] = [path_to_index]
+
+        xml = get_asset('scholarly-articles-items-response.xml')
+        requests_mock.get(f'{gateway}{path_to_index}', text=xml)
+
+        xml = get_asset('scholarly-articles-article1-response.xml')
+        requests_mock.get(f'{gateway}{path_to_index}/article1/versions/latest', text=xml)
+
+        es_mock = mocker.patch('search.populate.Elasticsearch.index')
```

Comment:
mocking a library method/function is a smell, but I expect this to be temporary as the `/populate` says it is. It opens a question for the future: do we use a real ElasticSearch during the test suite execution or do we mock it? The former option probably becomes necessary very quickly given the strong integration with it.

Reply:
My thoughts are; This test is checking that `/populates` executes correctly given the expected values and passes data to the `Elasticsearch.index` method in the correct format. Additional integration tests using a running ES instance should also be created but as this PR is getting a little long, I would prefer to do that separately.

Reply:
Yes, it's a general direction of the test suite for later (we don't even read from there at the moment)